src/Symfony/Component/Translation/Tests/Extractor/PhpExtractorTest.php

```diff
@@ -87,9 +91,9 @@ public function resourcesProvider()
         return [
             [$directory],
             [$phpFile],
-            [glob($directory.'*')],
+            [glob($directory.\DIRECTORY_SEPARATOR.'*')],
```

Comment:
It should work with "just" `/`, no? I mean, even on Windows.

Reply:
The functionality works with `/`, but the comparisons on line `73` and `74` do not.